vroom/selector.vroom

```diff
@@ -0,0 +1,98 @@
+Maktaba includes a selector utility to help plugin authors launch a window to
```

Comment:
+ license?

Reply:
Wait, didn't we determine it's not needed for vroom files? It's going to be really ugly at the top of every vroom file, and we don't need it on e.g. markdown files AFAIK.

Reply:
oh, that's probably true. =) Good to know.